sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRebaseDatetimeSuite.scala

```diff
@@ -134,33 +134,35 @@ abstract class ParquetRebaseDatetimeSuite
         tsOutputType: String = "TIMESTAMP_MICROS",
         inWriteConf: String = SQLConf.PARQUET_REBASE_MODE_IN_WRITE.key,
         inReadConf: String = SQLConf.PARQUET_REBASE_MODE_IN_READ.key): Unit = {
-      withTempPaths(2) { paths =>
-        paths.foreach(_.delete())
+      withAllParquetWriters {
+        withTempPaths(2) { paths =>
+          paths.foreach(_.delete())
         val oldPath = getResourceParquetFilePath("test-data/" + fileName)
         val path3_x = paths(0).getCanonicalPath
         val path3_x_rebase = paths(1).getCanonicalPath
-        val df = Seq.tabulate(N)(rowFunc).toDF("dict", "plain")
-          .select($"dict".cast(catalystType), $"plain".cast(catalystType))
-        withSQLConf(SQLConf.PARQUET_OUTPUT_TIMESTAMP_TYPE.key -> tsOutputType) {
+          val df = Seq.tabulate(N)(rowFunc).toDF("dict", "plain")
+            .select($"dict".cast(catalystType), $"plain".cast(catalystType))
+          withSQLConf(SQLConf.PARQUET_OUTPUT_TIMESTAMP_TYPE.key -> tsOutputType) {
```

Comment:
Something wrong with indentation here and below:
<img width="642" alt="Screenshot 2022-01-06 at 22 56 02" src="https://user-images.githubusercontent.com/1580697/148443305-f6890111-26c2-4538-9abe-53a28144c46b.png">